stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/ui/components/messageoptions/MessageOptions.kt

```diff
@@ -107,8 +110,18 @@ public fun defaultMessageOptionsState(
     val isOwnMessage = selectedMessageUserId == currentUser?.id
     val isUserMuted = currentUser?.mutes?.any { it.target.id == selectedMessageUserId } ?: false
     val isMessageSynced = selectedMessage.syncStatus == SyncStatus.COMPLETED
+    val isMessageFailed = selectedMessage.syncStatus == SyncStatus.FAILED_PERMANENTLY
 
     return listOfNotNull(
+        if (isOwnMessage && isMessageFailed) {
+            MessageOptionItemState(
+                title = R.string.stream_compose_resend_message,
+                iconPainter = painterResource(R.drawable.stream_compose_ic_resend),
```

Comment:
This icon is the same as the reply icon. Duplicated it in case somebody wants to override the icon independently.